pulsar-client/pom.xml

```diff
@@ -91,6 +91,12 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.avro</groupId>
+      <artifactId>avro</artifactId>
+      <version>1.8.2</version>
```

Comment:
Version should be defined in top-level pom

Reply:
ok